### PR TITLE
Prepare 3.30.0

### PR DESCRIPTION
**Pull request by davidmreed**
# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

**Review by jstvz — DISMISSED** [review]

**Review by davisagli — CHANGES_REQUESTED** [review]
@Br4nd0R is working on dependency updates

File: HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,38 @@
...
+  We recommend reformatting nested dependencies and discontinuing use of ``allow_newer`` and package uninstalls now to prepare for these future changes. 
+
+Changes:
+
+- We released a `new suite of documentation for CumulusCI <https://cumulusci.readthedocs.io/en/latest/>`_.
+- CumulusCI now caches org describe data in a local database to provide significant performance gains, especially in ``generate_dataset_mapping``.
+- The ``cci org browser`` command now has a ``--path`` option to open a specific page and a ``--url-only`` option to output the login URL without spawning a browser.
+- We improved CumulusCI's behavior when loading Record Type data from Snowfakery to avoid showing a warning to the user.
+- We improved messaging about errors while loading ``cumulusci.yml``.
+
```
Comment (davisagli) [inline HISTORY.rst:35]: We should mention the new version of Snowfakery and link to its release notes

File: HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,38 @@
...
+  All dependencies should be listed in install order. 
+  
+  We recommend reformatting nested dependencies and discontinuing use of ``allow_newer`` and package uninstalls now to prepare for these future changes. 
+
+Changes:
+
+- We released a `new suite of documentation for CumulusCI <https://cumulusci.readthedocs.io/en/latest/>`_.
+- CumulusCI now caches org describe data in a local database to provide significant performance gains, especially in ``generate_dataset_mapping``.
+- The ``cci org browser`` command now has a ``--path`` option to open a specific page and a ``--url-only`` option to output the login URL without spawning a browser.
+- We improved CumulusCI's behavior when loading Record Type data from Snowfakery to avoid showing a warning to the user.
```
Comment (davisagli) [inline HISTORY.rst:33]: I think we were already suppressing the warning.

File: HISTORY.rst [hunk 3]
```diff
@@ -2,6 +2,37 @@
...
+  
+  We recommend reformatting nested dependencies and discontinuing use of ``allow_newer`` and package uninstalls now to prepare for these future changes. 
+
+Changes:
+
+- We released a `new suite of documentation for CumulusCI <https://cumulusci.readthedocs.io/en/latest/>`_.
+- CumulusCI now caches org describe data in a local database to provide significant performance gains, especially in ``generate_dataset_mapping``.
+- The ``cci org browser`` command now has a ``--path`` option to open a specific page and a ``--url-only`` option to output the login URL without spawning a browser.
+- We improved messaging about errors while loading ``cumulusci.yml``.
+- CumulusCI now uses Snowfakery 1.6 (see its `release notes <https://github.com/SFDO-Tooling/Snowfakery/releases/tag/v1.6>`_).
```
Comment (davisagli) [inline HISTORY.rst:34]: Use a double underscore after the link here or we will not be able to make another link labeled `release notes` without reStructuredText throwing an error (yeah, I know).

**Review by davisagli — COMMENTED** [review]
Paul started adding pull request numbers at the end of each release note, which I kind of like.

File: HISTORY.rst (hunk 3 above)
Comment (davidmreed) [reply HISTORY.rst:34]: Good grief, RST!

**Review by davisagli — APPROVED** [review]

**Review by davisagli — APPROVED** [review]

**Pull request merged**